app/templates/admin/serviceTranscript.html

```diff
@@ -0,0 +1,150 @@
+{% extends "base.html"%}

```

Comment:
I'm not sure why you guys have service transcript changes in here. You weren't using this file, right?

Reply:
We did not work with this file. After a little investigating, it seems to be the result of a previous merge conflict where we kept the file not knowing it was no longer being used. There is a another serviceTranscript.html in the templates/main folder that is in development while this one, in templates/admin, is not.